rpcserver.go

```diff
@@ -2450,7 +2450,7 @@ func handleGetPeerInfo(s *rpcServer, cmd interface{}, closeChan <-chan struct{})
 			ID:             statsSnap.ID,
 			Addr:           statsSnap.Addr,
 			AddrLocal:      p.ToPeer().LocalAddr().String(),
-			Services:       fmt.Sprintf("%08d", uint64(statsSnap.Services)),
```

Comment:
I think we should just add a new field instead of changing the format. If anyone depends on this being the old format this would be a breaking change. Perhaps we have `Services` and `ServicesString` to denote the string representation?